### DISCUSSION TITLE
Code is missing in warning block on Installation page

### DISCUSSION THREAD

**Discussion by dejagersh** [livewire/livewire#6419] [7. Bugs (archived)]
![image](https://github.com/livewire/livewire/assets/9981548/97ce2eed-3908-4ba8-8a66-434523e3582c)

Looks like it's actually running that `@livewireScripts` directive 😁

<img width="1151" alt="image" src="https://github.com/livewire/livewire/assets/9981548/e01adb3f-6785-4c51-8624-b146bb94cdf0">

### Additional information

P.S. I wasn't sure where to give feedback on this docs page. The "issues" page says I need to make a PR on the `livewire/docs` repo but I wouldn't know how to fix it.

If it shouldn't be here, please let me know which medium to use and I'll move it.

**Comment by joshhanley** [comment]
@dejagersh thanks for reporting! Docs are now in the main repo here https://github.com/livewire/livewire/tree/main/docs

PRs to fix are welcome! 🙂

**Reply by dejagersh** [reply]
Would love to help, but this doesn't seem easy to fix for someone that is not familiar with the codebase. I tried to run the docs website locally, but it's unclear how to do that.
